Projects/Forever/Sources/Store/ForeverStore.swift

```diff
@@ -0,0 +1,146 @@
+import Apollo
+import Flow
+import Foundation
+import Presentation
+import UIKit
+import hCore
+import hGraphQL
+
+public final class ForeverStore: LoadingStateStore<ForeverState, ForeverAction, ForeverLoadingType> {
+    @Inject var octopus: hOctopus
+
+    public override func effects(
+        _ getState: @escaping () -> ForeverState,
+        _ action: ForeverAction
+    ) -> FiniteSignal<ForeverAction>? {
+        switch action {
+        case .fetch:
+            return FiniteSignal { callback in
+                let disposeBag = DisposeBag()
+                let query = OctopusGraphQL.MemberReferralInformationQuery()
+                disposeBag += self.octopus.client.fetch(query: query, cachePolicy: .fetchIgnoringCacheCompletely)
+                    .onValue { data in
+                        let grossAmount = data.currentMember.insuranceCost.monthlyGross
+                        let grossAmountMonetary = MonetaryAmount(
```

Comment:
When you switch to MoneyFragment in query you can just init with
 MonetaryAmount(fragment: grossAmount.fragments.moneyFragment) etc